lib/commons/aria/attributes.js

```diff
@@ -37,6 +37,24 @@ aria.validateAttr = function (att) {
 };
 
 /**
+ * Validate the value of an ARIA attribute with an idref
+ * @param  {HTMLElement} doc Root element
+ * @param  {HTMLElement} node The element to check
+ * @param  {String} attr The name of the attribute
+ * @param  {String} value The value of the attribute
+ * @return {Boolean}
+ */
+aria.validateIdrefType = function (doc, node, attr, value) {
```

Comment:
The logic looks excellent to me. What I'm wondering is if there's a better way of communicating what's going on to users. The proposed implementation doesn't give any sort of feedback about why in one case `aria-errormessage` is passed and in another it is failed. It'd be great if we could expose that somehow. I'm not too sure how we'd go about doing that though... Perhaps we could have a separate check for `aria-errormessage`? It could be a `none` check, that raises an error if none of these three cases are met. What do you think?

Reply:
So, if this fn throws and I include a test for `throws`...

```js
throw new Error('The value of aria-errormessage must be supported using an additional technique that announces the message (e.g., aria-live, aria-describedby, role=alert, etc.)')
```

Is that enough? Too much?

Reply:
The tests are passing. The only thing I haven't tried to do is:
> Perhaps we could have a separate check for aria-errormessage?

Given my lack of familiarity with the testing harness, I'm not sure how I would go about that. Is it possible to make that an additional feature?